src/layout/Stepper/StepperTheme.tsx

```diff
@@ -19,20 +27,26 @@ export interface StepperTheme {
 export const stepperTheme: StepperTheme = {
   base: 'grid grid-cols-[min-content_1fr] gap-x-3',
   step: {
-    base: 'border-l border-solid border-panel-accent translate-x-1/2',
+    base: {
+      common: 'border-l border-solid border-stroke-neutral-2 translate-x-1/2',
+      dot: 'mt-4',
+      circle: 'mt-10'
+    },
     marker: {
-      base: 'rounded-full w-[9px] h-[9px] bg-surface',
-      container:
-        'w-max pt-1 pb-0.5 backdrop-blur-md -translate-x-[calc(50%+0.5px)]',
-      active: 'bg-info',
+      base: 'rounded-full w-2.5 h-2.5 bg-content-assets-neutral-3',
+      container: {
+        common:
+          'text-badges-colors-outline-brand-text-default w-max relative bg-transparent -translate-x-[calc(50%+0.5px)]',
```

Comment:
probably shoudln't be a `badge` reference token here. will need to reach out to design probably